babel/README.md

```diff
@@ -0,0 +1,8 @@
+# Babel application
+
+Full documentation about it [here](https://mochajs.org/#-require-module-r-module)
```

Comment:
Might want another section here. I'm no Babel expert so feel free to modify accordingly...

```md
## Runtime Environment:

Three possibilities:
1. Avoid features which require runtime support (including new Built-ins and generators)
2. Require `@babel/runtime` directly, allowing usage of generators, but not polyfills which would modify the global environment
3. Require `@babel/polyfill` as a peer dependency and document that the consuming app must require it

This example app does not use features which require runtime support.
```

Reply:
Looks good. Will add.